sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -2201,15 +2194,13 @@ object GpuOverrides {
       ExprChecks.unaryProjectNotLambdaInputMatchesOutput(TypeSig.STRING, TypeSig.STRING),
       (a, conf, p, r) => new UnaryExprMeta[Upper](a, conf, p, r) {
         override def convertToGpu(child: Expression): GpuExpression = GpuUpper(child)
-      })
-      .incompat(CASE_MODIFICATION_INCOMPAT),
```

Comment:
If there are still some known issues, should this PR be removing them from the incompatible list?  I'm thinking the reason string needs to be updated to document the current known issues rather than point to an old, resolved cudf issue.

Reply:
Got it. Updated
I planned to add this new doc in another PR but yes, seems it is not a good idea to remove the incompat tag here.